src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/security-recommendations/[type]/page.tsx

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getServerSession } from "next-auth";
+import { redirect } from "next/navigation";
+import { SecurityRecommendationsLayout } from "../SecurityRecommendationsLayout";
+import { getSecurityRecommendationsByType } from "../securityRecommendationsData";
+import { authOptions } from "../../../../../../../../api/utils/auth";
+import { getSubscriberBreaches } from "../../../../../../../../functions/server/getUserBreaches";
+import { getLocale } from "../../../../../../../../functions/server/l10n";
+
+interface SecurityRecommendationsProps {
+  params: {
+    type: string;
+  };
+}
+
+export default async function SecurityRecommendations({
+  params,
+}: SecurityRecommendationsProps) {
+  const session = await getServerSession(authOptions);
+  const locale = getLocale();
+  if (!session?.user?.subscriber?.id) {
+    return redirect("/");
+  }
+  // TODO: Filter breaches for dataClassesEffected depending on
+  // security recommendation type.
+  const breaches = await getSubscriberBreaches(session.user);
+
+  const { type } = params;
+  const pageData = getSecurityRecommendationsByType(type);
+  if (!pageData) {
+    redirect("/redesign/user/dashboard");
+  }
+
+  return (
+    <SecurityRecommendationsLayout
+      label="Security recommendations"
```

Comment:
Q: Is this user facing, and need to be localized?

Reply:
Good catch — localized in [71c0a8e](https://github.com/mozilla/blurts-server/pull/3382/commits/71c0a8eb9157158a81157611de4d62debb5e67a2).